config/webpack.config.js

```diff
@@ -145,7 +145,7 @@ module.exports = env => {
           })
         },
         {
-          test: /(\.png|\.gif)$/,
+          test: /(\.png|\.gif|.svg)$/,
```

Comment:
Shouldn't there be a leading backward slash in front of the dot?

Reply:
You're right... fixed!